db/migrate/20150405012126_remove_approval_status_from_reservation.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveApprovalStatusFromReservation < ActiveRecord::Migration
+  def change
+    remove_column :reservations, :approval_status, :string
```

Comment:
We might want to touch this migration up a little bit: if someone with existing requested/denied reservations runs these migrations that data is going to be lost.